parachain/runtime/kintsugi/src/lib.rs

```diff
@@ -263,6 +263,26 @@ parameter_types! {
     pub const ExecutiveBody: BodyId = BodyId::Executive;
 }
 
+// Set the initial parameters for CollatorSelection.
+pub struct SetCollatorSelection;
+impl frame_support::traits::OnRuntimeUpgrade for SetCollatorSelection {
+    fn on_runtime_upgrade() -> frame_support::weights::Weight {
+        collator_selection::DesiredCandidates::<Runtime>::put(9);
+        collator_selection::CandidacyBond::<Runtime>::put(100 * KINT.one());
```

Comment:
Here are my proposed values, I think it makes sense for `DesiredCandidates` to be `9` since we have exactly that many "invulnerables". The open question on my end is if 100 vKINT makes sense for the `CandidacyBond`.

Reply:
Updated to 600 vKINT for the `CandidacyBond`.

Reply:
Setting it to 9 would mean that we would have up to 18 collators active, right? 9 invulnerables and 9 dynamically chosen ones?